.github/workflows/build-lint-test.yml

```
@@ -0,0 +1,55 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+name: Node.js CI - Build + Lint + Test
+
+on: pull_request
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Use Node.js v16
```

Let's use nodejs v20, 16 is not maintained anymore.

https://endoflife.date/nodejs

```suggestion
            - name: Use Node.js v20
```